gui_qt/bootstrap_report.py

```
@@ -73,21 +73,30 @@ def coerce_bool(value: Any, default: bool = False) -> bool:
-        "bootstrap_on_build": coerce_bool(rag.get("bootstrap_on_build"), True),
-    }
+def read_batch_context_flags(
+    config: dict[str, Any],
+    game_root: str | None = None,
+) -> dict[str, bool]:
+    """Effective batch context flags (global defaults + per-project overrides)."""
+    try:
+        from project_context_settings import resolve_batch_context_flags
+
+        return resolve_batch_context_flags(config, game_root)
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)